bodhi/tests/server/services/test_updates.py

```diff
@@ -3002,11 +2984,10 @@ def test_edit_update(self, *args):
 
         Karma has been reset.
         """).strip()
-        self.assertMultiLineEqual(up['comments'][-1]['text'], comment)
-        self.assertEqual(len(up['builds']), 1)
-        self.assertEqual(up['builds'][0]['nvr'], 'bodhi-2.0.0-3.fc17')
-        self.assertEqual(self.db.query(RpmBuild).filter_by(nvr='bodhi-2.0.0-2.fc17').first(),
-                         None)
+        assert_multiline_equal(up['comments'][-1]['text'], comment)
```

Comment:
what is the difference between this method from unittest and doing a basic ``assert foo == bar``?

Reply:
It's used for string with a couple of lines. In case the assert fails, it doesn't dump the whole strings at you but shows a diff.